nix/README.md

```diff
@@ -110,7 +78,17 @@ thing you have to do is run `dune build src/app/cli/src/mina.exe`. You
 can also just run `eval "$buildPhase"` to run the same command as
 would be run inside the nix sandbox.
 
```

Comment:
```suggestion
would be run inside the nix sandbox. Note that `opam` will **not** be available in that shell, since Nix takes over the job of computing and installing dependencies. If you need to modify the opam switch, use the impure build (next section).